meson.build

```diff
@@ -109,7 +109,8 @@ if extension_tool.found()
     'pack',
     '--force',
     '--out-dir=@0@'.format(builddir),
-    '--extra-source=@0@/metadata.json'.format(builddir)
+    '--extra-source=@0@/metadata.json'.format(builddir),
+    '--extra-source=@0@/dirs.js'.format(builddir),
```

Comment:
Would it be better to add `dirs.js` to `dist_files` around line 59 when it’s added to `sources`? Then the existing `foreach dist_files` loop can do its magic

Reply:
I tried this, but it didn't work. I realized then that it was the same problem of metadata.json, in which it can't handle subdirs correctly. I didn't spend time investigating it more, to be honest, and I don't want this PR to be blocked on this. Thanks for the review!